ui/views.py

```diff
@@ -22,3 +25,18 @@ class MediaView(apiviews.MediaItemMixin, generics.RetrieveAPIView):
     serializer_class = serializers.MediaItemPageSerializer
     renderer_classes = [TemplateHTMLRenderer]
     template_name = 'ui/media.html'
+
+
+class MediaAnalyticsView(apiviews.MediaAnalyticsView):
+    """
+    View for rendering an individual media item's analytics.
+    Extends the DRF's media item analytics view.
+
+    """
+    renderer_classes = [TemplateHTMLRenderer]
+
+    template_name = 'ui/analytics.html'
+
+    def get(self, request, pk):
+        response = super().get(request, pk)
+        return Response({'analytics_json': json.dumps(response.data)})
```

Comment:
Can we use the ``to_json`` filter instead? Otherwise the analytics will have the same XSS fail if there is a user-controllable string in them in the future.